apps/api/src/auth/discord.strategy.ts

```
@@ -0,0 +1,35 @@
+      callbackURL: process.env.DISCORD_REDIRECT_URI,
+      scope: ['identify', 'email'],
+      passReqToCallback: true,
+    });
+  }
+
+  async validate(
+    req: any,
+    accessToken: string,
+    refreshToken: string,
+    profile: Profile,
+    done: Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)